03_launch_mgmt_cluster.sh

```diff
@@ -291,7 +291,11 @@ function update_capm3_imports(){
   fi
 
   # Render the IPAM components from local repo instead of using the released version
-  ./hack/tools/bin/kustomize build "${IPAMPATH}/config/" > config/ipam/metal3-ipam-components.yaml
+  if [ "${CAPI_VERSION}" == "v1alpha3" ]; then
```

Comment:
nit: can we rather use `CAPM3_VERSION` here not to confuse ourselves in the future with different API versions and use the same var as in L118 of lib/common.sh ?

Reply:
in L118 we are using `CAPI_VERSION` var to differentiate IPAM branches, so I believe that was the reason why we were conditioning here using same variable.

Reply:
ah sorry, I wanted to mean L188

Reply:
but this is minor change, we can do it in a follow up PR too

Reply:
@fmuyassarov see #747